src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -438,9 +438,15 @@ CACHES = {
 }
 
 ELASTICSEARCH_HOSTS = [
-    {% for es_host in groups.es_coordinator -%}
-    '{{ es_host }}',
+{% if groups.es_coordinator %}
```

Comment:
If `[es_coordinator]` is missing from the inventory, will this throw an error or evaluate to something falsey? Alternatively, you could use instead of

```
{% for es_host in groups.es_coordinator -%}
```

something like the following:
```
{% for es_host in groups.es_coordinator|default(groups.elasticsearch) -%}
```

which I think does what we're trying to do here, without this extra `if`/`else` branching.

Reply:
Sure, Danny. Thank you.

Reply:
@dannyroberts Pushed changes as suggested. Verified again in a staging environment as well., working as expected. Please let me know if I'm missing anything. thanks